calculator/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
```

Comment:
Let's make some clearing....
![image](https://user-images.githubusercontent.com/31897434/91722389-151a0200-eb92-11ea-93af-d01bac5dcfa3.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/


You can just create a div with the name of the App you're about to build.